src/Units/KaijuWarsUnits.java

```diff
@@ -0,0 +1,783 @@
+          continue;
+        rezzer.resurrectionTypeMap.put(um, rezToModel);
+        break;
+      }
+    }
+
+    SuicideAttackTracker kaputnikTracker = StateTracker.instance(gi, SuicideAttackTracker.class);
+    for( UnitModel um : grms.unitModels )
+    {
+      KaijuWarsUnitModel umCast = (KaijuWarsUnitModel) um;
+      if( !umCast.suicideAttack )
+        continue;
+
+      kaputnikTracker.killTypeMap.add(um);
+    }
+  }
+
+  /*
+   * Units I do not plan to implement at this time
+
+   * Freezer (0-counter unit with 1 damage that slows targets by a tile; too complicated and bad)
+   * Food Cart (attracts kaiju within 3 tiles; not implementing kaiju path/targeting mechanics)
+   * Shark Jet (faster Fighter that moves after shooting; messy and either OP or UP)
+   * Cannon of Boom (immobile Rocket equivalent that teleports between labs; ew)
+   */
+
+  /*
+   * Boosts from technologies and tactics; consider making expensive variant units with these boosts
+
+   * tank: +2 counter while on urban
+   * missile: +2 ATK while stationary, 1-2 scoot-n-shoot
+   *  - Giving them the former
+   *  - The latter is hilariously OP, so let's make an "uber missile" that does that?
+   * AA: all attacks get +1 damage for each AA next to target, force air units to land on hit
+   *  - Added the +1; the other is too jank
+   * inf: +2 counter on ROUGH terrain
+   * Radar: +2 science if you end turn within 2 tiles of a Kaiju
+   *  - Should implement as +2 CO energy, but only for opposing Kaiju
+   * bomber: enable hitting air for 1 damage
+   * fighter: +2 counter on empty(plains?)/water
+   *  - Adding
+   * ground: +2 ATK hills, +2 counter on forest/jungle
+   *  - Adding the first
+   * air: +3 move on airport
+   *  - Adding
+
+   * Hover Drives: Gives all ground +1 move and the ability to move on water (Super Z2 doesn't benefit)
+   * Ultimate Autobahn: Gives all ground +2 move if starting on road/bridge
+   */
+
+  /*
+   * Special mechanics
+
+   * slowed by rough terrain: missiles, maser, food cart
+   * slow on counter: tank, Liger Panther, AA, OGR(air+ground)
+   * +2 counter if you don't move: inf
+   * Bushplanes put out Crises/fires adjacent to the tile they die
+   *  - Now are capture units
+   * Police prevent Crises/fires from showing up within 2 tiles
+   *  - Now are capture units
+   * Radar counters certain Kaiju abilities while within 2 tiles
+   *  - piercing vision, will add those
+   * Copters and Sky Carrier give +1 ATK to adjacent allies
+
+   * Freezer slows on attack
+   * Guncross Wing becomes Guncross Robot on death
+   * Super Z2 also lives on death, but just goes from 4 to 1 counter power
+   * Super Z2 can go on water, but is a land unit for all other purposes
+   * Kaputnik has a suicide attack
+   * Shark Jet can move (but not shoot) again after shooting
+   * Sky Carrier lets you build air units adjacent to it
+   * Cannon Of Boom cannot move normally, but can teleport between labs as its turn
+   */
```

Comment:
More dev notes? Didn't you decide not to add Freezer and Cannon? At least according to prior comments.

Reply:
Yeah. I did decide that. Same with Shark Jet.
Freezer (and Shark Jet) is complicated, and really bad in a world with counterattacks.
Cannon is hard to say how you would even implement it (I guess you could say "it's the only unit Ports build" instead of Kaiju, but ehhhh)